### ISSUE TITLE
Since classifier is not enabled, why have classifier parameters?

### ISSUE THREAD

**Issue by rhyolight** [numenta/NAB#234]
In the [NuPIC model params](https://github.com/numenta/NAB/blob/master/nab/detectors/numenta/modelParams/model_params.json#L21-L27):

```javascript
        "clEnable": false,
        "clParams": {
            "alpha": 0.035828933612158,
            "clVerbosity": 0,
            "regionName": "CLAClassifierRegion",
            "steps": "1"
        },
```

Since `clEnable` is false, can't you just remove the whole `clParams` block?

**Comment by BoltzmannBrain** [comment]
Yes, the `clParams` dict can be removed, but the `clEnable` entry must remain as it is expected by the CLA model. Including this there are a few issues around nab/nupic params (#231, #235), so I'll wrap them together into a future PR.

**Comment by BoltzmannBrain** [comment]
Closed w/ #236
